obkvrpc/connection_pool.go

```diff
@@ -37,27 +44,55 @@ type PoolOption struct {
 	databaseName string
 	userName     string
 	password     string
+
+	maxConnectionAge     time.Duration
+	enableSLBLoadBalance bool
 }
 
 type ConnectionPool struct {
 	option *PoolOption
 
 	connections []*Connection
 	rwMutexes   []sync.RWMutex
+	connMgr     *ConnectionMgr
+}
+
+type ConnectionLifeCycleMgr struct {
```

Comment:
I guess we could move all SLB/LiftCycle code into a new file(s).

Reply:
done